lib/msal-electron-proof-of-concept/src/appConfig/clientApplicationBase.ts

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { ApplicationConfiguration, buildApplicationConfiguration } from './applicationConfiguration';
+
+/**
+ * ClientApplicationBase abstract class
+ *
+ * Abstract class that provides a base for Public and Private Application classes.
+ */
+export abstract class ClientApplicationBase {
+    private _config: ApplicationConfiguration;
+
+    constructor(applicationConfiguration: ApplicationConfiguration) {
+        this._config = buildApplicationConfiguration(applicationConfiguration);
```

Comment:
Please try to avoid underscore. we are following the latest convention in TS where we do not add underscore for private methods.

Reply:
@technical-boy - consider "borrowing" a tslint.json from a repo that enforces these rules.

Reply:
Example https://github.com/microsoft/vscode/blob/master/tslint.json

Reply:
@bgavrilMS We'll be putting together a comprehensive eslint ruleset for this repo soon, so it's okay if Hector's code isn't perfect in this regard.

Reply:
I actually grabbed the tslint config from msal-core. I thought it best to keep the monorepo's consistency over super strict linting rules for just the msal-electron part.